src/api/app/models/workflow.rb

```diff
@@ -47,27 +50,27 @@ def steps
   end
 
   def filters
-    filters = {}
-
-    return filters if supported_filters.blank?
+    return {} if filters_configuration.blank?
```

Comment:
I don't like this change from `supported_filters` to `filters_configuration`. We use the term `supported` everywhere in the workflow feature. I agree that the code in this method is now a bit confusing due to the `if/else`, but renaming `supported_filters` to `filters_configuration` doesn't improve the code.

Reply:
Okay, I have no super string feelings about this.